### PR TITLE
[SE-5144] Implement Header Component (prototype)

### PR DESCRIPTION
**Pull request by keithgg**
## Description

Implements the changes required for the new OpenCraft edX theme. The changes in this repo are contained to HTML changes. Replacing components with their react-bootstrap counterparts so that they can render correctly.

This has only been tested for Learner's under the Courses section so far. Only the Monotone theme has been implemented. Changing to the other themes should not require any further modification of this repo.

## Supporting information

- https://gitlab.com/opencraft/dev/open-edx-theme/-/issues/7
- https://tasks.opencraft.com/browse/SE-5144

## Testing instructions

None yet.

## Deadline

None

## Other information

Include anything else that will help reviewers and consumers understand the change.
- A similar PR should be open in the open-craft/frontend-app-learning repo.

### CODE REVIEW

File: src/learning-header/AuthenticatedUserDropdown.jsx [hunk 1]
```diff
@@ -1,50 +1,61 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
```
Comment (mtyaka) [inline src/learning-header/AuthenticatedUserDropdown.jsx]: Were these changes made automatically by your editor? EdX JS styleguide says to prefer single quotes for strings, and we usually try to stick to the edX styles.

**Comment by mtyaka** [comment]
Nice work @keithgg! I thought more changes would be required, but I'm glad to see I was wrong.

File: src/learning-header/AuthenticatedUserDropdown.jsx (hunk 1 above)
Comment (keithgg) [reply src/learning-header/AuthenticatedUserDropdown.jsx]: You're right. My bad. I'll revert those.

**Comment by keithgg** [comment]
Closing PR as it's no longer needed.

**Pull request closed without merging**